src/org/gk/database/AttributeEditConfig.java

```diff
@@ -38,6 +38,10 @@
 	private static AttributeEditConfig config;
 	// The uneditable attributes list
 	private List uneditableAttributes;
+	// The uncreatable class list
+	private List uncreatableClasses;
+	// The uncreatable class contact
+	private String uncreatableClassesContact;
```

Comment:
Change to: private String uncreatableClassesContact = "Peter D'Eustachio"; // Default to Peter